train.py

```diff
@@ -483,6 +485,8 @@ def train(hyp, opt, device, tb_writer=None, wandb=None):
     parser.add_argument('--project', default='runs/train', help='save to project/name')
     parser.add_argument('--name', default='exp', help='save to project/name')
     parser.add_argument('--exist-ok', action='store_true', help='existing project/name ok, do not increment')
+    parser.add_argument('--model-output-dir', default=None, help='output the best model to this directory')
```

Comment:
I would be a bit more explicit in the comment that the reason for this is to provide a consistent output location, otherwise you can do this by wrangling the `--project` and `--name` arguments.